rts/Lua/LuaUnsyncedRead.cpp

```diff
@@ -2725,14 +2725,23 @@ int LuaUnsyncedRead::GetCameraNames(lua_State* L)
 
 /***
  * @function Spring.GetCameraState
- * @param useReturns false
- * @return CameraState cameraState
- */
-/***
- * @function Spring.GetCameraState
- * @param useReturns true? (Default: `true`) Return multiple values instead of a table.
- * @return CameraName name
- * @return any ... depend on current controller mode.
+ * @param useReturns [boolean] (optional, default: true)
+ *   If true or omitted, returns a table with all camera state fields. If false, returns the camera name and a variable number of camera-specific values as separate return values.
+ *
+ * @return table cameraState (if useReturns is true or omitted)
+ *   Table with the following fields:
+ *     - name: string (camera controller name)
+ *     - [other fields]: number (camera state fields, depend on controller)
+ *
+ * @return string name, ... (if useReturns is false)
+ *   The camera controller name, followed by a variable number of values depending on the current camera mode:
```

Comment:
If you want to document this on the variadic returns overload, you can just add the documentation in the previous format on the specific return value.

Reply:
I understand, I was not familiar with the doc format and it looked a bit odd to me, but I understand better now. I'll update the doc format like it was before.